### DISCUSSION TITLE
附加模块MFA总是有更新提示，是不是不推荐手动升级它？

### DISCUSSION THREAD

**Discussion by gsxgmpkt9020** [MAA1999/M9A#289] [Q&A] [answered]
看了下手动升级MFA似乎还要改配置文件。

如果可以不去手动升级它的话，觉得对于普通用户可以关闭这个MFA有新版本的提示

**Comment by Windsland52** [comment, marked as answer]
一般来说不用管mfa的更新，等m9a更新的时候会自动打包新版的mfa。
mfa不手动更新不影响用户使用m9a

**Reply by gsxgmpkt9020** [reply]
> 一般来说不用管mfa的更新，等m9a更新的时候会自动打包新版的mfa。 mfa不手动更新不影响用户使用m9a

配置文件里面能否加个提示开关？

**Reply by Windsland52** [reply]
似乎是mfa新版的功能，等m9a下次发版就能用了
